Resources/views/Collector/db.html.twig

```diff
@@ -1,23 +1,45 @@
 {% extends app.request.isXmlHttpRequest ? 'WebProfilerBundle:Profiler:ajax_layout.html.twig' : 'WebProfilerBundle:Profiler:layout.html.twig' %}
 
+{% set profiler_markup_version = profiler_markup_version|default(1) %}
```

Comment:
this needs to be done inside the ``toolbar`` block too, because the rendering of the toolbar does not render the full template but only the toolbar block